posthog/management/commands/generate_demo_data.py

```diff
@@ -78,9 +78,10 @@ def handle(self, *args, **options):
                     print(
                         "Master project reset!"
                         if options["reset_master"]
-                        else f"Demo data ready! Log in as {email} with password {password}.\n"
+                        else "\nDemo data ready!\n\n"
                         "If running DEMO mode locally, log in instantly with this link:\n"
-                        f"http://localhost:8000/signup?email={email}"
+                        f"http://localhost:8000/login?email={email}\n"
```

Comment:
When you run PostHog with `DEMO=1`, there's password-less login, which is unified with signup, which is why the instant login link is `/signup?email={email}`. In regular mode (without `DEMO=1`), signup is indeed `/login`, but those are separate cases. So this PR breaks the former case and adds handling for the latter case.
To support both use cases, we should keep the existing `/signup?email={email}` link and add a new `/login?email={email}&password={password}` one.

Reply:
Thanks for all the info and getting me up to speed here. I've now checked the former case and found that it shouldn't have been working in the first place. Let me know if I'm misunderstanding something. Otherwise I've added a fix for this case as well.